lib/pliny/librato/metrics/backend.rb

```diff
@@ -54,23 +54,20 @@ def start_timer
           end
         end
 
-        def start_counter
-          @counter = Thread.new do
-            loop do
-              msg = metrics_queue.pop
-              msg == POISON_PILL ? break : enqueue_librato(msg)
+        def flush_librato
+          sync do
+            $stderr.puts "pliny-librato: counter_cache.flush_to: %s" % counter_cache.instance_variable_get(:@cache).inspect
```

Comment:
I'm assuming the change is this https://github.com/librato/librato-metrics/pull/137?